app/src/main/java/zegreanu/cristi/geoquiz/Question.kt

```diff
@@ -1,3 +1,34 @@
 package zegreanu.cristi.geoquiz
 
-data class Question(val textResId: Int, val answerTrue: Boolean)
+import android.os.Parcel
+import android.os.Parcelable
+
+data class Question(val textResId: Int, val answerTrue: Boolean, var isCheated: Boolean = false) :
```

Comment:
You might want to explore with https://proandroiddev.com/parcelable-in-kotlin-here-comes-parcelize-b998d5a5fcac - not sure if it's stable or not.